org-clubhouse.el

```diff
@@ -1030,6 +1030,16 @@ which labels to set."
                     "\n")
        ""))))
 
+(defun org-clubhouse-headline-from-my-tasks (level)
+  "Prompt my active stories and create a single `org-mode' headline at LEVEL."
+  (interactive "*nLevel: \n")
+    (if org-clubhouse-username
+        (let* ((story-list (org-clubhouse--search-stories (format "owner:%s !is:done !is:archived" org-clubhouse-username)))
+               (stories (to-id-name-pairs story-list)))
+          (org-clubhouse-headline-from-story-id level
+                                                (find-match-in-alist (ivy-read "Select Story: " (-map #'cdr stories)) stories)))
+    (warn "Can't fetch my tasks `org-clubhouse-username' is unset")))
```

Comment:
1. this should probably be `user-error`
2. missing an "if" ("can't fetch my tasks *if*)

Reply:
I copied the warn from line 1166. If you want in some other way can you show my how. 

I squashed the fix with the rows now shorter.

Reply:
ah, yeah, that's fair. I'll make the change project-wide if I decide to.